src/layouts/components/Homepage/AnnouncementBody.tsx

```diff
@@ -0,0 +1,224 @@
+                            <Input
+                              placeholder="Learn more"
+                              id={`announcement-${announcementIndex}-link_text`}
+                              value={announcementLinkText}
+                              onChange={onChange}
+                            />
+                            <FormErrorMessage>
+                              {
+                                errors.announcementItems[announcementIndex]
+                                  .link_text
+                              }
+                            </FormErrorMessage>
+                          </FormControl>
+                          <FormControl
+                            isInvalid={
+                              !!errors.announcementItems[announcementIndex]
+                                .link_url
+                            }
+                          >
+                            <FormLabel>Link URL</FormLabel>
+                            <Input
+                              placeholder="Insert /page-url or https:  "
+                              id={`announcement-${announcementIndex}-link_url`}
+                              value={announcementLinkUrl}
+                              onChange={onChange}
+                            />
+                            <FormErrorMessage>
+                              {
+                                errors.announcementItems[announcementIndex]
+                                  .link_url
+                              }
+                            </FormErrorMessage>
+                          </FormControl>
+                          <Button
+                            id={`announcement-${announcementIndex}-delete`}
+                            onClick={() =>
+                              onDelete(
+                                `announcement-${announcementIndex}-delete`,
+                                "announcement"
+                              )
+                            }
+                            alignSelf="center"
+                            variant="clear"
+                            colorScheme="critical"
+                          >
+                            Delete announcement
+                          </Button>
+                        </Editable.Section>
+                      </Editable.DraggableAccordionItem>
+                    )
+                  }
+                )}
+              </Editable.Section>
+            </Editable.EmptySection>
+          </Editable.Accordion>
+        </Editable.Droppable>
+      </DragDropContext>
+      <Button
+        id={`announcement-${announcementItems.length}-create`}
+        onClick={() => onCreate({ target: { id: "announcement" } })}
+        variant="outline"
+        w="full"
+        leftIcon={<BiPlus fontSize="1.5rem" />}
+        isDisabled={announcementItems.length >= MAX_ANNOUNCEMENTS}
+      >
+        Add Announcement
```

Comment:
Might need to check with design on this text copy ("Add announcement" or "Add new announcement").

Reply:
Checked with @sehyunidaaa, it is indeed `Add announcement`